packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.stories.ts

```diff
@@ -0,0 +1,155 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { withActions } from '@storybook/addon-actions/decorator';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnNumberTextTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableActions,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { tableColumnTextTag } from '../../text';
+import { NumberTextFormat } from '../types';
+
+const simpleData = [
+    {
+        firstName: 'Homer',
+        lastName: 'Simpson',
+        age: 45.2358734623,
+        favoriteNumber: 0.0000002358967325
```

Comment:
Pretty sure Homer's favorite number is π, because "Mmmmmm....piiiiii".
```suggestion
        favoriteNumber: π
```